set.mm

```diff
@@ -31713,22 +31736,21 @@ generally appear in a single form (either definitional, but more often
 
   ${
     $d x A $.
-    $( Two ways to say " ` A ` is a set":  A class ` A ` is a member of the
-       universal class ` _V ` (see ~ df-v ) if and only if the class ` A `
-       exists (i.e. there exists some set ` x ` equal to class ` A ` ).
-       Theorem 6.9 of [Quine] p. 43. _Notational convention_:  We will use the
-       notational device " ` A e. _V ` " to mean " ` A ` is a set" very
-       frequently, for example in ~ uniex .  Note that a class ` A ` which is
-       not a set is called a _proper class_.  In some theorems, such as
-       ~ uniexg , in order to shorten certain proofs we use the more general
-       antecedent ` A e. V ` instead of ` A e. _V ` to mean " ` A ` is a set."
-
-       Note that a constant is implicitly considered distinct from all
-       variables.  This is why ` _V ` is not included in the distinct variable
-       list, even though ~ df-clel requires that the expression substituted for
-       ` B ` not contain ` x ` .  (Also, the Metamath spec does not allow
-       constants in the distinct variable list.)  (Contributed by NM,
-       26-May-1993.) $)
```

Comment:
I think this explanation is unnecessary, but since it's the first class constant to be introduced in this database: ok, I will add it and propose something inspired from the existing and your proposals.